src/pages/Beach/[beachid]/[beach_name]/Edit/index.js

```diff
@@ -0,0 +1,15 @@
+import Layout from "components/Layout/Layout";
```

Comment:
can you make the folder name here lowercase?

it's a little annoying to do because mac filesystem is case insentitive, so you'll need to change the folder name to `edit2`, commit that to git, and then change it to `edit` and then commit that